doc/CommitStyle.md

```diff
@@ -0,0 +1,19 @@
+# Commit message style #
+
+To make the commit messages consistent, the following style of messages is recommended:
+```
+  <type>: [<section>] Your message here [Fix #<num>] (credits) ```
+with
+  * **`<type>`** Type of change, see the list below. **Always include this.**
+  * **optional `[<section>]`** if the commit is in a special section, for example graphics or build system.
```

Comment:
I figured that we'd have "sections" at some point (buildsystem, rcdgen, etc come to mind)
There was a branch part of this doc (came before <type>) but i removed that